redaxo/src/addons/install/pages/index.php

```diff
@@ -12,18 +12,17 @@
     static $markdown;
 
     if (!$markdown) {
-        $parser = rex_markdown::factory();
-
-        // prior to 5.10 rex_markdown did not prevent xss
-        $stripTags = rex_string::versionCompare(rex::getVersion(), '5.10.0-dev', '<');
-
-        $markdown = static function (string $content) use ($parser, $stripTags): string {
-            if ($stripTags) {
-                $content = strip_tags($content);
-            }
-
-            return $parser->parse($content);
-        };
+        if (rex_string::versionCompare(rex::getVersion(), '5.10.0-dev', '<')) {
```

Comment:
könnte man den fallback fall einfach unit-testen?

Reply:
dann müsste ich aus der anonymen funktion hier eine richtige machen (oder eine methdeo) und sie woanders platzieren.

Und so ganz stimmen würde der test aber auch nicht, da die parsedown-version in 5.10 nicht die gleiche ist wie in 5.7.